zerver/tests/test_message_fetch.py

```diff
@@ -4533,6 +4563,37 @@ def test_has_reaction(self) -> None:
         self.assert_length(messages, 1)
 
 
+class MessageIsTest(ZulipTestCase):
+    def test_message_is_followed(self) -> None:
```

Comment:
Cordelia and Hamlet automatically follow topics when they send a message, so one could imagine making this test a bit more robust by having some followed topics and some not and testing as such. See the is resolved test for an example.

Also, as a prep commit, it might make sense to move that is resolved test to this new test class?

Reply:
> Cordelia and Hamlet automatically follow topics when they send a message.

When I send a message from hamlet or cordelia there is no entry in the `UserTopic` table indicating that they've followed the topic. Is there some other table that indicates if a user is following a topic other than `UserTopic`?

Reply:
Ah, I was misreading the comments in `test_user_topics.AutomaticallyFollowTopicsTests`, but you could maybe do something similar here. If nothing else, I'd confirm via a test that a message in a topic that's not followed isn't returned.